### ISSUE TITLE
Power Save Mode for battery sense does not work

### ISSUE THREAD

**Issue by Flynnn** [mccloudaero/evpr#7]
The current solution is nonsense, replace with better solution.

**Comment by Flynnn** [comment]
Fixed. b334c16

High-Side control required to avoid defeating the purpose of the dividers by driving them high during power save mode.

So, should we use PFET or NFET for the high-side control? High-side control requires the introduction of a third FET in order to level shift up to the control voltage.

PFET is not practical as a level shifter (I'm sure you could figure it out, but it won't be simple), whereas NFET works great. The caveat is NFET must have source tied to ground.

We want high-side gates to default to OFF. If we pick NFET for them, their gates must be biased to ground. But this will make the level shifter useless since it shorts to ground. So, the high-side gates must be PFET.

In conclusion, power save uses two high-side PFETS driven by an NFET level shifter.
